tests/utils/constants/constants.go

```diff
@@ -23,6 +23,7 @@ var (
 	OCMEnv                    = "OCM_ENV"
 	RhcsClusterProfileENV     = "RHCS_PROFILE_ENV"
 	ClusterTypeManifestDirEnv = "CLUSTER_ROSA_TYPE"
+	ClusterIDEnv              = "CLUSTER_ID"
```

Comment:
May I know the meaning of the env?

Reply:
A, this is duplication of Shelly's commit
It was added to Test initialization step (that runs before the tests) by Shelly in previous PR
https://github.com/terraform-redhat/terraform-provider-rhcs/blob/main/tests/e2e/init_test.go#L18
So the Test Case knows the cluster ID to use

Reply:
OK. Let's keep it in current PR. I will enhance this part in next PR to make clusterID comes from the manifest file output rather than env. Of course we will also accept the env variable if it set to convenient the testing work.